Scala/Scala.sublime-syntax

```diff
@@ -9,12 +9,12 @@ scope: source.scala
 contexts:
   main:
     - include: storage-modifiers
-    - include: keywords
     - include: declarations
     - include: inheritance
     - include: imports
     - include: comments
     - include: block-comments
+    - include: keywords
```

Comment:
Adding the new keywords this high can prevent other patterns from matching, e.g. I believe your matching of `new` would prevent `initialization` from matching its expected `new` keyword.

I’d also recommend keeping the same scope for keywords that you’re matching without context now, e.g. `type` is `keyword.declaration.scala` in `declarations`.

It’s a pity the scopes have diverged from the TextMate grammar so much…